database/factories/GameAchievementSetFactory.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Database\Factories;
+
+use App\Models\AchievementSet;
+use App\Models\Game;
+use App\Models\GameAchievementSet;
+use App\Platform\Enums\AchievementSetType;
+use Illuminate\Database\Eloquent\Factories\Factory;
+
+/**
+ * @extends Factory<GameAchievementSet>
+ */
+class GameAchievementSetFactory extends Factory
+{
+    protected $model = GameAchievementSet::class;
+
+    public function definition(): array
+    {
+        return [
+            'game_id' => Game::factory(),
+            'achievement_set_id' => AchievementSet::factory(),
+            'type' => AchievementSetType::Core,
+            'title' => null,
+            'order_column' => 0,
+        ];
+    }
+
+    public function type(string $type): self
+    {
+        return $this->state(function (array $attributes) use ($type) {
+            return [
+                'type' => $type,
+                'title' => $type === AchievementSetType::Core ? null : $this->faker->words(3, true),
+                'order_column' => $type === AchievementSetType::Core ? 0 : $this->faker->numberBetween(1, 10),
```

Comment:
This field/column is weirdly named. Are you sure this is how it was meant to be used?

Reply:
Yes. There are TODOs in all models with a `DisplayOrder` field to rename that field to `order_column`.

`order_column` for `GameAchievementSet` will likely just represent the order in which each set is rendered on a game page, similar to the ordering of achievements.